aom/src/main/java/com/nedap/archie/aom/OperationalTemplate.java

```diff
@@ -20,17 +30,78 @@
  */
 @XmlAccessorType(XmlAccessType.FIELD)
 @XmlType(name="OPERATIONAL_TEMPLATE")
-@XmlRootElement(name="archetype")
+@XmlRootElement(name="operational_template")
 public class OperationalTemplate extends AuthoredArchetype {
 
     /**
      * terminology extracts from subarchetypes, for example snomed codes, multiple choice thingies, etc
      */
-    @XmlElement(name="terminology_extracts") //TODO: requires an adapter for JAXB to work
+    @XmlTransient
     private Map<String, ArchetypeTerminology> terminologyExtracts = new ConcurrentHashMap<>();//TODO: is this correct?
-    @XmlElement(name="component_terminologies") //TODO: requires an adapter for JAXB to work
+    @XmlTransient
     private Map<String, ArchetypeTerminology> componentTerminologies = new ConcurrentHashMap<>();
 
+    @XmlElement(name="terminology_extracts")
+    private List<XmlArchetypeTerminology> xmlTerminologyExtracts;
+    @XmlElement(name="component_terminologies")
+    private List<XmlArchetypeTerminology> xmlComponentTerminologies;
+
```

Comment:
todo: mark these two fields @JsonIgnore and possibly transient

Reply:
Jackson seems to already ignore these fields as there are no public getters.